### PR TITLE
Add padding to frontmatter

### PR DESCRIPTION
**Pull request by crashmoney**
- add whitespcae between top of page and metadata frontmatter
- improve file title alignment for Sliding Panes

### CODE REVIEW

File: obsidian.css [hunk 1]
```diff
@@ -2642,6 +2642,30 @@ iframe {
...
+  border-radius: 0;
+  border: solid var(--quote-opening-modifier);
+  border-width: 0px 0px 0px 2px;
+  background-color: transparent;
+  color: var(--green);
+  font-style: italic;
+  padding: 0 0 0 calc(var(--nested-padding) / 2);
+  margin-inline-start: var(--nested-padding);
+}
+
```
Comment (colineckert) [inline obsidian.css:2668]: Thanks! I really like the better spacing on the frontmatter, will definitely keep that. 

This blockquote selector appears to be a duplicate of styling beneath this. Please remove.
```suggestion

```

File: obsidian.css [hunk 2]
```diff
@@ -4099,6 +4123,10 @@ body.plugin-sliding-panes-rotate-header {
 body.plugin-sliding-panes-rotate-header .view-header-title:before {
   display: none;
 }
+/* better vertically align file title */
+body.plugin-sliding-panes-rotate-header.plugin-sliding-select-orientation-mixed .workspace > .mod-root > .workspace-leaf > .workspace-leaf-content > .view-header {
+  padding: 2.5px;
+}
```
Comment (colineckert) [inline obsidian.css:4129]:
```suggestion
```
Tested this change locally and I prefer the current alignment. This change adds slightly too much space underneath the note title (to the left between the edge of the pane) and misaligns the title with the note icon above it. 

Thanks, but please omit this change and I'll approve the PR with just the frontmatter updates :)

**Comment by colineckert** [comment]
Adding updates locally along with others and pushing. Thanks so much for opening this PR!

**Comment by crashmoney** [comment]
Thank you!

Since the theme has been updated, should I close this PR?

**Comment by colineckert** [comment]
Appreciate it!

**Pull request closed without merging**